lib/TeleIrc.js

```diff
@@ -193,6 +212,18 @@ class TeleIrc {
         return from.toLowerCase() === name.toLowerCase();
       });
 
+      if ((this.config.irc.nickservService.length != 0) &&
```

Comment:
This is pretty much just pedantics, but typically for things like this I like to check for greator than 0, instead of != 0. If you look down on line 227, we're using that check (though we're using the inverse).